03 Writing Algorithms/20 Historical Data/11 Rolling Window/12 Examples.php

```diff
@@ -1 +1,310 @@
+<p>The following examples demonstrate some common practices for implementing rolling windows.</p>
+
+<h4>Example 1: Price Actions</h4>
+<p>Using <code>RollingWindow</code> to cache the last 3 trade bars, the following algorithm could identify volume contraction breakout price action pattern and buy SPY accordingly to ride on the capital inflow. We take 2% profit and stop loss at 1%.</p>
```

Comment:
This strategy is the same of the resource "take-profit-stop-loss-example,html" added in this PR. It should be imported.